components/dropdown/dropdown-content-mixin.js

```diff
@@ -475,6 +475,8 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 
 	async __openedChanged(newValue) {
 
+		await this.updateComplete;
```

Comment:
I know this is already merged so I wouldn't worry too much about this, but in the absence of a test to cover the error case, I might be inclined to add a code comment even though we like the code to speak for itself.  An unfamiliar dev might think to remove this line and it might seem ok if all tests pass, and it's not obvious from the code what erroneous behaviour might result.